packages/@coorpacademy-components/src/molecule/teams-popin/test/fixtures/loading-background.js

```diff
@@ -3,7 +3,7 @@ import Login from './login';
 
 export default {
   props: defaultsDeep(Login.props, {
-    backgroundImageUrl: 'https://static-staging.coorpacademy.com/content/BG_teams@2x.jpg',
+    backgroundImageUrl: 'https://static-staging.coorpacademy.com/content/teams/BG_teams@2x.jpg',
```

Comment:
@StefanoA1 des urls ont été changés sur AWS ?

Reply:
@jomaora oui (je ne sais pas quand/pourquoi) mais j'ai noté ça en cherchant des fixtures qui prennent des assets côté S3, donc j'ai mis à jour les fixtures